src/App.tsx

```diff
@@ -1,101 +1,125 @@
+} from './utils/redirects';
 import { IaTjenesterMetrikkerContextProvider } from './metrikker/IaTjenesterMetrikkerContext';
 import VedlikeholdSide from './FeilSider/Vedlikehold/VedlikeholdSide';
-import SamtalestøttePodletpanel from './Forside/Samtalestøttepanel/SamtalestøttePodletpanel';
-import { amplitudeClient } from './amplitude/client';
-import { useOrgnr } from './utils/orgnr-hook';
+import {
+    getEkstradata,
+    SykefraværAppData,
+    useSykefravarsstatistikk,
+} from './hooks/useSykefravarsstatistikk';
+import { AnalyticsClient } from './amplitude/client';
+import { useAnalytics } from './amplitude/useAnalytics';
 
-const App: FunctionComponent = () => {
+interface Props {
+    analyticsClient: AnalyticsClient;
+    samtalestøttePodlet?: React.ReactNode;
+}
 
-    sendEventDirekte('forside', 'sidelastet');
+const App: FunctionComponent<Props> = ({ analyticsClient, samtalestøttePodlet }) => {
+    useAnalytics(analyticsClient);
     return (
-        <BrowserRouter basename={BASE_PATH}>
-            <AltinnOrganisasjonerProvider>
-                <AltinnOrganisasjonerMedTilgangTilStatistikkProvider>
-                    <VirksomhetsdataProvider>
-                        <EnhetsregisteretProvider>
-                            <SummertSykefraværshistorikkProvider>
-                                <SykefraværshistorikkProvider>
-                                    <FeatureTogglesProvider>
-                                        <IaTjenesterMetrikkerContextProvider>
-                                            <main id='maincontent'>
-                                                <AppContent />
-                                            </main>
-                                        </IaTjenesterMetrikkerContextProvider>
-                                    </FeatureTogglesProvider>
-                                </SykefraværshistorikkProvider>
-                            </SummertSykefraværshistorikkProvider>
-                        </EnhetsregisteretProvider>
-                    </VirksomhetsdataProvider>
-                </AltinnOrganisasjonerMedTilgangTilStatistikkProvider>
-            </AltinnOrganisasjonerProvider>
-        </BrowserRouter>
+        <IaTjenesterMetrikkerContextProvider>
+            <main id="maincontent">
+                <AppContent
+                    {...useSykefravarsstatistikk()}
+                    analyticsClient={analyticsClient}
+                    samtalestøttePodlet={samtalestøttePodlet}
+                />
+            </main>
+        </IaTjenesterMetrikkerContextProvider>
     );
 };
 
-const AppContent: FunctionComponent = () => {
+export const AppContent = ({
+    altinnOrganisasjoner,
+    altinnOrganisasjonerMedStatistikk,
+    summertSykefravær,
+    fraværshistorikk,
+    virksomhetsdata,
+    analyticsClient,
+    enhetsInformasjon,
+    samtalestøttePodlet,
+}: SykefraværAppData & {
+    analyticsClient?: AnalyticsClient;
+    samtalestøttePodlet?: React.ReactNode;
```

Comment:
For å ikke være avhengig av samtalestøttepodlet skal rendres i test tas den heller inn som en optional prop